rmw_zenoh_cpp/src/detail/rmw_context_impl_s.cpp

```diff
@@ -179,21 +179,21 @@ class rmw_context_impl_s::Data final
           "rmw_zenoh_cpp", "[rmw_context_impl_s] z_call received an invalid reply.\n");
       }
     }
-
-    // Initialize the shm manager if shared_memory is enabled in the config.
-    shm_provider_ = std::nullopt;
-#ifndef _MSC_VER
```

Comment:
Should we retain the Windows check? Does the SHM feature work on Windows?

Reply:
SHM for Windows is supported and works :)

Reply:
Okay. Hope we can pass all the tests on Windows. :smiley: